app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -260,6 +260,12 @@ export default {
         return {
           builder: {
             premium: false,
+            basic: {
+              components: {
+                // add 'simplefile' file upload component to formBuilder in advanced mode
+                simplefile: this.userType !== this.ID_MODE.PUBLIC
+              }
```

Comment:
Should we be doing this? I know this fileUpload isn't exactly just a "simple component", but this also asks questions about how unified our custom components vs standard components should be -> mixing both could cause quite a bit of confusion in the long run for our users.

Reply:
What is this doing? We don't add the orgbook manually for example but it's in both basic and advanced.

Reply:
Agreed. i guess the file upload component should be re-named from 'simplefile' to 'fileupload'
the same way the OrgBook component is just called 'orgbook'
seems out of scope for this ticket.